vector/src/main/res/values/strings.xml

```diff
@@ -2509,6 +2509,7 @@
     <string name="login_splash_text2">Keep conversations private with encryption</string>
     <string name="login_splash_text3">Extend &amp; customise your experience</string>
     <string name="login_splash_submit">Get started</string>
+    <string name="login_splash_already_have_account">I already have an account</string>
```

Comment:
Just to complete our IRL discussion, https://github.com/vector-im/element-android/blob/main/vector/src/main/res/values/strings_login_v2.xml exists and so do https://github.com/vector-im/element-android/blob/main/vector/src/main/res/values/strings_login_v2.xml#L27. The resource from this file are not synced with Weblate, so feel free to do whatever you want with it, like removing strings for instance.